Add map "Alpine Assault" to ladder

endless-creation opened the pull request:
Submitting this 1v1 map I made for consideration and incorporation into the ladder.

endless-creation committed:
Add map "Alpine Assault" to ladder

endless-creation committed:
Fix an issue with map Alpine Assault

Near the top-left oil derrick, Ore Miners can get stuck on initial spawn when building an Ore Refinery there.  Adding a rock to the offending location prevents the miner from spawning in a bad spot

alexp8 commented on package/INI/MPMaps.ini:
```diff
@@ -8336,4 +8337,16 @@ Waypoint3=49036
+Author=endless
+Description=[2] Alpine Assault
+EnforceMaxPlayers=True
+GameModes=Standard
```
which gamemode should this map be in?

endless-creation replied:
Battle, I suppose? How should I modify that in the map itself? Or would I Modify `MPMaps.ini` directly?  

Wouldn't mind if it got added to the ladder as well although I'm not sure if that's a separate process.

alexp8 replied:
if you update the map file with Battle game mode, the tool will pick that up and apply it

for this map specifically go ahead and manually update mpmaps.ini, or update the map file and re-run the tool and confirm it gets updated

endless-creation replied:
Updated

alexp8 commented on package/Maps/Yuri's Revenge/2_alpine_assault.png:
you can exclude the starting spot boxes from the map preview, the client populates those

endless-creation replied:
Updated

alexp8 commented on package/Maps/Yuri's Revenge/2_alpine_assault.map:
```diff
@@ -0,0 +1,3127 @@
+
+[PreviewPack]
+1=BIACcgAEwBtAMnRABAAaQCSANMAVQASAAnIABMAbQDJ0QAQAGkAkgDTAFUAEgAJyAATAG0
+2=yAsAIAXQ5PDQ5PDQ6JQATAEE6PDQ4PDI4JgBTAFEAkgAJyAATAG0AydEAEABpAJIA0wBVA
```
the preview pack looks pretty small?

endless-creation replied:
There's no preview on this map file (I selected no preview in Final Alert when saving to keep the file size small). I figured cncnet client adds a preview regardless.  Should I add the preview that Final Alert does automatically?

alexp8 replied:
yeah include the default preview, since that is what is shown in the loading screen, else it's black

endless-creation replied:
Updated

alexp8 commented:
if the map has precaptured tech lets include that in the map name

endless-creation committed:
Fix map Alpine Assault for cncnet

- Remove starting positions from preview
- Change game mode to Battle
- Add "(Precaptured)" to map title
- Add standard map preview to map file

endless-creation commented:
> if the map has precaptured tech lets include that in the map name

Updated map title

alexp8 commented on package/Maps/Yuri's Revenge/2_alpine_assault.map:
```diff
@@ -136,7 +630,7 @@ PlayerControl=no
 
 [FA2spVersionControl]
-Version=881
+Version=884
```
what is this version #? for your own reference?

endless-creation replied:
FA2SP auto-increments this number when you manually save the map, or when it auto-saves.
